 
 <launch>


  <arg name="use_rviz"                 default="true"/>
  

  

  <include file="$(find pckg)/launch/xsturret_gazebo.launch" >
    <arg name="robot_model"                       value="wxxms"/>
    <arg name="use_rviz"                 default="$(arg use_rviz)"/>
    <arg name="paused"                            default="false"/>
 </include>
 

    <node
        name="pan_tilt"
        pkg="pckg"
        type="pan_tilt"
        respawn="false"
        output="screen"
    />


    <node
        name="tracker"
        pkg="pckg"
        type="coord_tracker"
        respawn="false"
        output="screen"
    />

    
	<node pkg="octomap_server" type="octomap_server_node" name="octomap_server" output="screen" launch-prefix="xterm -e">
		<param name="resolution" value="0.05" />
		
		<!-- fixed map frame (set to 'map' if SLAM or localization running!) -->
		<param name="frame_id" type="string" value="world" />
		
		<!-- maximum range to integrate (speedup!) -->
		<param name="sensor_model/max_range" value="5.0" />
		
		<!-- data source to integrate (PointCloud2) -->
		<remap from="cloud_in" to="/camera1/depth/points" />
	
	</node>


    

    
    

 
    

    
</launch>


